Cargo.toml

```diff
@@ -124,9 +124,9 @@ ledger-transport-hid = "0.10.0"
 libc = "0.2.97"
 libloading = "0.7.2"
 linkme = "0.3.24"
-# branch = "main"
-masp_primitives = { git = "https://github.com/anoma/masp", rev = "3aacc707c5948e7423589ac617305448bead9842" }
-masp_proofs = { git = "https://github.com/anoma/masp", rev = "3aacc707c5948e7423589ac617305448bead9842", default-features = false, features = ["local-prover"] }
+# branch = "murisi/receive-randomness-with-borsh2"
```

Comment:
Can we merge this to main?

Reply:
Yes, I've just merged this to `main`. Now updating this branch...